src/spaceObjects/beamEffect.cpp

```diff
@@ -28,6 +29,7 @@ BeamEffect::BeamEffect()
     beam_texture = "beam_orange.png";
     beam_fire_sound = "sfx/laser_fire.wav";
     beam_fire_sound_power = 1;
+    fire_ring = true;
```

Comment:
fire_ring requires a memberReplication as well then, as the clients need to know it for proper rendering.

Reply:
Ah, missed that, tested only in a single Instance. While re-testing, I noticed that lifetime is also only known in the server. Will fix both.

Reply:
As the lifetime is simulated on the client as well, and updated on the server every tick, it's better to specify an update interval as 2nd parameter to the registerMemberReplication, like "100" for 100ms between updates. Else it will send out data every frame.

Reply:
I see. But are you sure that parameter is measured in milliseconds? On other occurances, it seem to vary between 0.1 and 1.0, which looks like it is seconds. So 100ms would be 0.1.

Reply:
My mistake. Been working with other code that does things in ms a lot.

Reply:
Okay. So it should now in order then, I guess.